src/app/services/comment.service.ts

```diff
@@ -7,15 +7,19 @@ export class CommentService {
   constructor() { }
 
 //finds a user once using the userId
-  findUser(userId) {
+  userRef(userId) {
```

Comment:
This is right. Although, on second thought, getUserRef() would be more appropriate. Really good method naming is hard. I like to call any methods that return a specific thing "getThing()" and since we're actually returning a reference to the user's spot in the DB I suppose "getUserRef()" is more appropriate than simply "userRef()" This is my own mistake.